scripts/datasets/DET.py

```diff
@@ -0,0 +1,203 @@
+"""Prepare DET(Object detection) datasets"""
+import os
+import argparse
+import tarfile
+from os.path import join, isdir
+import glob
+import xml.etree.ElementTree as ET
+import json
+import time
+import sys
+from gluoncv.utils.filesystem import try_import_cv2
+from concurrent import futures
+from gluoncv.utils import download, makedirs
+cv2 = try_import_cv2()
+def parse_args():
+    """dataset parameter."""
+    parser = argparse.ArgumentParser(
+        description='Download DET dataset and prepare for tracking')
+    parser.add_argument('--download-dir', type=str, default='~/.mxnet/datasets/DET/',
+                        help='dataset directory on disk')
+    parser.add_argument('--instanc-size', type=int, default = 511, help='instanc image size')
```

Comment:
I think `instanc` is a typo, right? should be `instance`. 

And there is no space around `=`. `default = 511` should be `default=511`.